tasklist/webapp/src/main/java/io/camunda/tasklist/webapp/security/identity/DefaultUserGroupService.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.tasklist.webapp.security.identity;
+
+import static io.camunda.tasklist.property.IdentityProperties.FULL_GROUP_ACCESS;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Component
+@Profile("!consolidated-auth")
```

Comment:
🔧 Given the setup is based on configuration classes only and component scans are deactives, I think you can omit the profile here, it should actually not have any effect.
```suggestion
```